app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -86,3 +84,11 @@
     </li>
   <% end %>
 </ul>
+
+<div class="component-markdown">
+  <h2 class="component-doc-h2">If you cannot find a suitable component</h2>
```

Comment:
I think this whole section could be resumed to one line, since the page we link to has the principles and conventions at the top of the page and explains when to build a new component.

```suggestion
  <p class="govuk-body">If you cannot find a suitable component consider extending an existing component or <a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/develop-component.md" class="govuk-link">creating a new one</a></p>.
```

As mentioned in a previous comment, I have concerns about promoting the usage of Design System styles as it can easily become a gateway to a non-componentised frontend.

Reply:
My hope was that by putting the link to the Design System styles as the very last thing on the page that developers would use it as their last resort, but I take your point, so I've made the change above.